internal/netif/netif.go

```diff
@@ -50,7 +57,21 @@ func (m *netifManagerDefault) EnsureIPAddress() error {
 
 	l, err := m.LinkByName(m.devName)
 	if err != nil {
-		return xerrors.Errorf("could not get loopback interface %v", err)
+		var linkNotFoundErr netlink.LinkNotFoundError
+		if !errors.As(err, &linkNotFoundErr) && !m.manageInterface {
```

Comment:
The condition is too complicated and wrong. In case `m.manageInterface == true` this will never lead to a return, but we actually only want to continue in case the device was not found.